### PR TITLE
Give example of append key to stringslice

### PR DESCRIPTION
**Pull request by dereknola**
We mention using the `+` in a config but don't give a good example if it.

### CODE REVIEW

**Review by brandond — COMMENTED** [review]
This is great, we did need an example.

I'm not sure if there's a good place to call this out, but when using the `key+` syntax, the order of the files does matter - for example:

```yaml
# test1.yaml
node-taint:
  - alice=bob:NoExecute
```
```yaml
# test2.yaml
node-taint+:
  - charlie=delta:NoSchedule
```
```yaml
# test3.yaml
node-taint:
  - eve=frank:NoExecute
```

The value from the 3rd file will clobber the combined values from the first two, as the files are loaded in lexicographic order, and appending happens when the file is loaded - it does not forever taint that key as an accumulating key. Maybe it should, I don't know.

**Comment by dereknola** [comment]
I think this is pretty clear from the documentation wording already, we say:

>  By default, configuration files are read from /etc/rancher/k3s/config.yaml and /etc/rancher/k3s/config.yaml.d/*.yaml in alphabetical order. The last value for a given key will be used. A + can be appended to the key to append the value to the existing string or slice, instead of replacing it.

Maybe I could reword it to:
```
By default, configuration files are read from /etc/rancher/k3s/config.yaml and /etc/rancher/k3s/config.yaml.d/*.yaml in alphabetical order. 

By default, the last value found for a given key will be used. A `+` can be appended to the key to append the value to the existing string or slice, instead of replacing it.
All subsequent entries of this key will also require a `+` to prevent overwriting the entire value.
```

**Review by brandond — APPROVED** [review]

**Review by briandowns — APPROVED** [review]
